.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: 'ecma-262'
+
+on: [pull_request, push]
+
+jobs:
+  build:
+    name: 'build & lint ecmarkup'
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-20.04
```

> Ubuntu-20.04 is ready to be the default version for the “ubuntu-latest” label in GitHub Actions and Azure DevOps. This change will be rolled out over a period of several weeks beginning on November, 30. (December, 22 - January, 4 will be a migration break due to holidays)

from https://github.com/actions/virtual-environments/issues/1816

Reply:
![image](https://user-images.githubusercontent.com/3842474/103707308-565f6a00-4fe9-11eb-8b75-16aa7e8d818a.png)

Reply:
the ubuntu version should never make a difference for the majority of node project's CI, there's no benefit in changing from the default.